src/Avalonia.NameGenerator.Tests/InitializeComponent/InitializeComponentTests.cs

```diff
@@ -13,26 +12,17 @@ namespace Avalonia.NameGenerator.Tests.InitializeComponent
     public class InitializeComponentTests
     {
         [Theory]
-        [InlineData(InitializeComponentCode.NamedControl, View.NamedControl, false)]
-        [InlineData(InitializeComponentCode.NamedControls, View.NamedControls, false)]
-        [InlineData(InitializeComponentCode.XNamedControl, View.XNamedControl, false)]
-        [InlineData(InitializeComponentCode.XNamedControls, View.XNamedControls, false)]
-        [InlineData(InitializeComponentCode.NoNamedControls, View.NoNamedControls, false)]
-        [InlineData(InitializeComponentCode.CustomControls, View.CustomControls, false)]
-        [InlineData(InitializeComponentCode.DataTemplates, View.DataTemplates, false)]
-        [InlineData(InitializeComponentCode.SignUpView, View.SignUpView, false)]
-        [InlineData(InitializeComponentCode.AttachedProps, View.AttachedProps, false)]
-        [InlineData(InitializeComponentCode.FieldModifier, View.FieldModifier, false)]
-        [InlineData(DevToolsCode.NamedControl, View.NamedControl, true)]
-        [InlineData(DevToolsCode.NamedControls, View.NamedControls, true)]
-        [InlineData(DevToolsCode.XNamedControl, View.XNamedControl, true)]
-        [InlineData(DevToolsCode.XNamedControls, View.XNamedControls, true)]
-        [InlineData(DevToolsCode.NoNamedControls, View.NoNamedControls, true)]
-        [InlineData(DevToolsCode.CustomControls, View.CustomControls, true)]
-        [InlineData(DevToolsCode.DataTemplates, View.DataTemplates, true)]
-        [InlineData(DevToolsCode.SignUpView, View.SignUpView, true)]
-        [InlineData(DevToolsCode.AttachedProps, View.AttachedProps, true)]
-        [InlineData(DevToolsCode.FieldModifier, View.FieldModifier, true)]
+        [InlineData(InitializeComponentCode.NamedControl, View.NamedControl, true)]
+        [InlineData(InitializeComponentCode.NamedControls, View.NamedControls, true)]
+        [InlineData(InitializeComponentCode.XNamedControl, View.XNamedControl, true)]
+        [InlineData(InitializeComponentCode.XNamedControls, View.XNamedControls, true)]
+        [InlineData(InitializeComponentCode.NoNamedControls, View.NoNamedControls, true)]
+        [InlineData(InitializeComponentCode.CustomControls, View.CustomControls, true)]
+        [InlineData(InitializeComponentCode.DataTemplates, View.DataTemplates, true)]
+        [InlineData(InitializeComponentCode.SignUpView, View.SignUpView, true)]
+        [InlineData(InitializeComponentCode.AttachedProps, View.AttachedProps, true)]
+        [InlineData(InitializeComponentCode.FieldModifier, View.FieldModifier, true)]
+        [InlineData(InitializeComponentCode.ControlWithoutWindow, View.ControlWithoutWindow, true)]
```

Comment:
Why remove the `DevToolsCode`/`InitializeComponentCode` separation? We can generate `this.AttachDevTools()` **only if** `Avalonia.Diagnostics` is present, that's the reason why I had to split the expected outputs to `DevToolsCode` and `InitializeComponent` code. The `DevToolsCode` contains `this.AttachDevTools()`, `InitializeComponentCode` doesn't. Are we _still_ unit testing the conditional `this.AttachDevTools()` generation?

Reply:
Yes, I suggest leaving the Avalonia.Diagnostics add modulation

Reply:
From what I can see, the tests that covered the conditional `this.AttachDevTools` generation are removed.
Where's the replacement?

Reply:
Now with these four cases, we cover all cases with DevTools

```C#
  [InlineData(InitializeComponentCode.AttachedPropsWithDevTools, View.AttachedProps, true)]
  [InlineData(InitializeComponentCode.AttachedProps, View.AttachedProps, false)]
  [InlineData(InitializeComponentCode.ControlWithoutWindow, View.ControlWithoutWindow, true)]
  [InlineData(InitializeComponentCode.ControlWithoutWindow, View.ControlWithoutWindow, false)]
```

For all other cases, AttachDevTools modulation is disabled, this covers InitializeComponent

```C#
        [InlineData(InitializeComponentCode.NamedControl, View.NamedControl, false)]
        [InlineData(InitializeComponentCode.NamedControls, View.NamedControls, false)]
        [InlineData(InitializeComponentCode.XNamedControl, View.XNamedControl, false)]
        [InlineData(InitializeComponentCode.XNamedControls, View.XNamedControls, false)]
        [InlineData(InitializeComponentCode.NoNamedControls, View.NoNamedControls, false)]
        [InlineData(InitializeComponentCode.CustomControls, View.CustomControls, false)]
        [InlineData(InitializeComponentCode.DataTemplates, View.DataTemplates, false)]
        [InlineData(InitializeComponentCode.SignUpView, View.SignUpView, false)]
        [InlineData(InitializeComponentCode.FieldModifier, View.FieldModifier, false)]
```